packages/client/src/pages/home/home.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link, Outlet } from 'react-router-dom'
+
+const Home = () => {
+  return (
+    <main>
+      <ul>
+        <li>
+          <Link to="/">Главная страница</Link>
+        </li>
+        <li>
+          <Link to="/login">Авторизация</Link>
+        </li>
+        <li>
+          <Link to="/signup">Регистрация</Link>
+        </li>
+        <li>
+          <Link to="/profile">Профиль</Link>
+        </li>
+        <li>
+          <Link to="/leaderboard">Таблица лидеров</Link>
+        </li>
+        <li>
+          <Link to="/forum">Форум</Link>
+        </li>
+        <li>
+          <Link to="/game">Игра</Link>
+        </li>
```

Comment:
[Предложение]
Продолжение прошлого. Если будет массив объектов, то по нему нужно пройтись и помапить в ссылку:
```suggestion
        {
          links.map({ path, title } => {
            return (
              <li><Link to={path}>{title}</Link></li>
            )
          })
        }
```

Reply:
сделано